doc/classes/RenderingServer.xml

```diff
@@ -4272,6 +4272,8 @@
 		</constant>
 		<constant name="LIGHT_PARAM_TRANSMITTANCE_BIAS" value="19" enum="LightParam">
 		</constant>
+		<constant name="LIGHT_PARAM_INTENSITY" value="20" enum="LightParam">
+		</constant>
```

Comment:
Please describe the property if you are able, probably good hint from the relevant property in light